### PR TITLE
Fix `.drone.yml` signature

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title.

### CODE REVIEW

**Comment by s0undt3ch** [comment]
Fixes https://github.com/saltstack/salt-bootstrap/issues/1395

File: .drone.yml [hunk 1]
```diff
@@ -1182,6 +1182,6 @@ depends_on:
 
 ---
 kind: signature
-hmac: 906ceab2f0ec44b6e9c99f698feeb4874c503fd9db2a8525ed2916445069c403
+hmac: 6f9e82fb6d1fc4166a76b998fecb456cb5e931f5c2914d023008e30fba1c8822
```
Comment (vutny) [inline .drone.yml:1185]: I got it. The maintainer needs to sign-off the modified YAML file each time it changes.

**Review by vutny — COMMENTED** [review]
How about `.drone.jsonnet`, is it being in use at all?

File: .drone.yml (hunk 1 above)
Comment (s0undt3ch) [reply .drone.yml:1185]: Yup

**Review by vutny — APPROVED** [review]
Alright, so the `.drone.jsonnet` is the template source of truth for CI, and the `.drone.yml` need to be generated and signed from it for changes to be propagated.

I'm thinking about describing this procedure in README or so...

**Comment by vutny** [comment]
@s0undt3ch All green, let's merge it!

**Pull request merged**